src/rank_llm/demo/rerank_inline_hits.py

```diff
@@ -87,4 +87,6 @@
 Path(f"demo_outputs/").mkdir(parents=True, exist_ok=True)
 writer.write_in_jsonl_format(f"demo_outputs/rerank_results.jsonl")
 writer.write_in_trec_eval_format(f"demo_outputs/rerank_results.txt")
-writer.write_ranking_exec_summary(f"demo_outputs/ranking_execution_summary.json")
+writer.write_infernece_invocations_history(

```

Comment:
Seems this typo is in all locations so my last comment about it maybe replace all

Reply:
thanks for catching, fixed now!